node/src/components/event_stream_server/tests.rs

```diff
@@ -1171,13 +1171,19 @@ async fn should_limit_concurrent_subscribers() {
 /// `resources/test/sse_data_schema.json` across different versions of the codebase.
 #[test]
 fn schema() {
+    // To generate the contents to replace the input JSON files, run the test
+    // and print the `actual_schema`  by uncommenting the `println!`
+    // towards the end of the test.
     let schema_path = format!(
         "{}/../resources/test/sse_data_schema.json",
         env!("CARGO_MANIFEST_DIR")
     );
     let expected_schema = fs::read_to_string(schema_path).unwrap();
     let schema = schema_for!(SseData);
     let actual_schema = serde_json::to_string_pretty(&schema).unwrap();
+
+    // println!("{}", actual_schema);
```

Comment:
Do we need this?

Reply:
Then maybe add a comment explaining that :) I would remove that comment if I had seen it :)

Reply:
There is a comment explaining that just above, at the beginning of the test.

Reply:
My brain would not go to the beginning of the test in search of explanation why a commented out `println!()` should not be removed.